.github/workflows/build-and-test.yml

```diff
@@ -108,7 +108,7 @@ jobs:
   unittest:
     strategy:
       matrix:
-        go-version: [1.18, 1.17]
+        go-version: [1.19, 1.18, 1.17]
```

Comment:
Don't we remove N-2 simultaneously we adding the new version N?

Reply:
I figured it would be easier to add 1.19 and make necessary changes to pass. Then update the 'required' check to 1.19 and 1.18 (removing 1.17) before removing 1.17. I don't have a strong preference though.